.travis.yml

```diff
@@ -0,0 +1,43 @@
+# Travis CI (MIT license) configuration file for Beans.
+# @link https://travis-ci.org/
+
+# For use with the Beans Theming Framework
+# @link https://github.com/Getbeans/Beans
+
+# This version does not include sniffers. Why? The codebase would fail.
+# Once we get the codebase compliant, then we'll update this file.
+
+sudo: false
+dist: trusty
+language: php
+
+env:
+  global:
+    - WP_VERSION=master WP_MULTISITE=0
+
+matrix:
+    fast_finish: true
+
+    include:
+        - php: 5.2
+          dist: precise
+        - php: 5.6
+        - php: 7.0
+        - php: 7.1
+
+    allow_failures:
+        - php: 7.0
```

Comment:
Why should PHP 7.1 be allowed to fail?

Reply:
@GaryJones Are you thinking no `allow_failures`?

Reply:
Allow failures on `nightly`, maybe HHVM, if you care to test for that (I wouldn't bother).

See https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/blob/02da0322dab24ba65bf1483136b375dc9706f21c/.travis.yml#L53-L56